libs/core/src/components/breadcrumb/breadcrumb.component.ts

```diff
@@ -0,0 +1,59 @@
+import { html } from 'lit'
+import { property } from 'lit/decorators.js'
+import { classMap } from 'lit/directives/class-map.js'
+
+import { GdsElement } from '../../gds-element'
+import { tokens } from '../../tokens.style'
+import { gdsCustomElement } from '../../utils/helpers/custom-element-scoping'
+import {
+  withLayoutChildProps,
+  withMarginProps,
+  withSizeXProps,
+} from '../../utils/mixins/declarative-layout-mixins'
+import { IconChevronLeft } from '../icon/icons/chevron-left.component'
+import BreadcrumbCSS from './breadcrumb.styles'
+
+@gdsCustomElement('gds-breadcrumb', {
```

Comment:
Missing doc comment for the class. I think this is also why no API-table is rendered in Storybook.

Reply:
Fixed it somehow i had removed the comment! 

Table also shows now 👌
![CleanShot 2025-05-16 at 14 10 54@2x](https://github.com/user-attachments/assets/c1fb6624-7ae4-4213-8904-5a68d24cc2ec)